niflow_manager/data/templates/base/spec.yml

```diff
@@ -0,0 +1,50 @@
+#  # requirement for the environment needed to run the analyses
```

Comment:
there is no need for the double `#` for comments

Reply:
i know, but it's easier to uncomment everything (everything but true comments)

Reply:
ok. as long as the template that's created runs through `nfm {test|build|install}` with this commented spec i'm ok.

`nfm init` followed by `nfm {test|build|install}` should all work without any changes from the user.

Reply:
no, it does not... the `build` requires `requirements` and `test` requires `test` together wit some workflow example.

perhaps, I didn't understand the idea of template in the same way. Do you want the template to be "runnable" without any changes?

Reply:
whatever is created with `nfm init <pkg-name> --language python` should be a working "python" project. so the spec inside it should be a working spec.

Reply:
ok, I changed the template, but also had to change testkraken that can take an empty test list (didn't want to include tests in the template) and fix a bug in pydra